src/app/main/component/events/components/event-details/event-details.component.html

```diff
@@ -6,12 +6,14 @@
         <a [routerLink]="routedFromProfile ? [backRoute, { tabId: 2 }] : [backRoute]" class="button-link">
           <div class="button-content">
             <img [src]="icons.arrowLeft" aria-hidden="true" alt="" class="button-arrow-img arrow" />
-            <div class="button-text">{{ 'homepage.events.btn.back-route' | translate }}</div>
+            <div class="button-text" *ngIf="false">{{ 'homepage.events.btn.back-route' | translate }}</div>
+            <div class="edit-news back-to-edit" *ngIf="true">{{ 'create-event.back-edit' | translate }}</div>
+            <button class="primary-global-button button-publish" *ngIf="true">{{ 'create-event.publish' | translate }}</button>
           </div>
         </a>
       </div>
 
-      <div *ngIf="role === roles.ORGANIZER || role === roles.ADMIN" class="edit-buttons">
+      <div *ngIf="role === (roles.ORGANIZER || role === roles.ADMIN) && false" class="edit-buttons">
```

Comment:
&& false??

Reply:
i changed all true and false to variables